jwst/ramp_fitting/ramp_fit.py

```diff
@@ -303,92 +328,237 @@ def ols_ramp_fit(model, buffsize, save_opt, readnoise_model, gain_model,
 
             opt_res.cr_mag_seg[num_int, :, rlo:rhi, :] = \
                        data_diff * (dq_cr != 0)
- 
-            m_by_var_int[num_int, rlo:rhi, :] = m_by_var.reshape(sect_shape)
-            inv_var_int[num_int, rlo:rhi, :] = inv_var.reshape(sect_shape)
 
-        slope_int[num_int, :, :] = \
-            utils.calc_slope_int(slope_int, m_by_var_int, inv_var_int, num_int)
+    # Compute the final 2D array of differences.
+    median_diffs_2d /= n_int
+    med_slopes = median_diffs_2d.copy()
```

Comment:
`median_diffs_2d` isn't used again, so it could be deleted to save space.

Reply:
Right.